package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@aligent/serverless-access-roles-generator",
+  "version": "0.0.1",
+  "description": "A Serverless framework plugin for creates roles to access resources",
```

Comment:
Typo - should be `Serverless framework plugin for creating roles to access resources`

Normally wouldn't bother but this is open source :sweat: